cdk/README.md

```diff
@@ -0,0 +1,130 @@
+# Welcome to ConsoleMe CDK Deployment
+
+## Motivation
+
+ConsoleMe is a web service that makes AWS IAM permissions and credential management easier for end-users and cloud administrators.
+
+Recently, I started to work on a new project and found the need to deploy ConsoleMe in a way that it would be available for multiple users, authenticating with the corporate authentication tools and scaled without managing servers.
+
+## Architecture
+
+The main idea is to use serverless services in order to remove the need from managing servers.
+
+![ConsoleMe on ECS Architecture](architecture.png "ConsoleMe on ECS Architecture")
+
+## Usage
+
+### Pre-requisites
+
+- Domain name managed with a public hosted zone on AWS Route 53.
+  Please collect this information and fill the `config.yaml` file with the hosted zone name and hosted zone id from Route 53.
```

Comment:
Also instruct them to generate strong random values for  `admin_temp_password` and `jwt_secret`

Reply:
Just pushed a fix